apps/admin/app/api/partners/top-connector/route.ts

```diff
@@ -0,0 +1,13 @@
+import { getLastWeek } from '@packages/dates/utils';
+
+import { respondWithTSV } from 'lib/nextjs/respondWithTSV';
+import { getScoutsForTopConnectorPartner } from 'lib/partners/getScoutsForTopConnectorPartner';
+
+export const dynamic = 'force-dynamic';
+
+export async function GET() {
+  const lastWeek = getLastWeek();
+  const rows = await getScoutsForTopConnectorPartner({ week: lastWeek });
+
+  return respondWithTSV(rows, `partners-export_supersim_${lastWeek}.tsv`);
```

Comment:
update the filename: "export_top_connectors"  and i dont think u need to export a week at a time. it coould just be an export for the last 14 days or soemthing?

Reply:
Done.